be/src/exec/exec_node.cpp

```diff
@@ -172,15 +172,13 @@ Status ExecNode::prepare(RuntimeState* state) {
     DCHECK(_runtime_profile.get() != NULL);
     _rows_returned_counter =
         ADD_COUNTER(_runtime_profile, "RowsReturned", TUnit::UNIT);
-    _memory_used_counter =
-        ADD_COUNTER(_runtime_profile, "MemoryUsed", TUnit::BYTES);
```

Comment:
You'd better remove this field if it's not used. Btw, `_memory_used_counter` seems to be used by `HashJoinNode::close`, is it really ok to remove it?

Reply:
Hi, as we can see. this counter ```_memory_used_counter``` is only used for count mem used for ```_tuple_pool``` and ```_hash_tbl```. 
But the ```_tuple_pool``` and ```_hash_tbl``` was already track the mem used by ```MemTracker``` and counted mem used in ```PeakMemoryUsage``` counter.  We do really need this counter ```_memory_used_counter```?

![image](https://user-images.githubusercontent.com/10553413/80563597-0d98ef80-8a1e-11ea-88a1-7d6f9b058ee3.png)


So, I think keep the ```_memory_used_counter ```  named ```MemoryUsed``` is confusing. 

I have two different suggestions

* Delete all code use ```_memory_used_counter``` in the future, all mem used is tracked by ```MemTracker```, we show peakmemusage it by the counter in ```MemTracker```.
* Keep the counter ```_memory_used_counter```, but change the name ```MemoryUsed``` to ```MemoryUsedByHashBTAndTuplePool```.

Which suggestion do you think is more suitable？@gaodayue

Reply:
>Delete all code use _memory_used_counter in the future, all mem used is tracked by MemTracker, we show peakmemusage it by the counter in MemTracker.

I prefer this.